lib/buffer.js

```diff
@@ -3,12 +3,12 @@
 
 const binding = process.binding('buffer');
 const { isArrayBuffer } = process.binding('util');
+const { deprecate } = require('internal/util');
 const bindingObj = {};
 const internalUtil = require('internal/util');
 
-class FastBuffer extends Uint8Array {}
+const FastBuffer = (() => class Buffer extends Uint8Array {})();
```

Comment:
Ok, here is the testcase: `Buffer.alloc(10).map(x => x + 1)` (just for future reference).